api/compaction_manager.cc

```diff
@@ -61,6 +74,31 @@ void set_compaction_manager(http_context& ctx, routes& r) {
         });
     });
 
+    cm::get_pending_tasks_by_table.set(r, [&ctx] (std::unique_ptr<request> req) {
+        return ctx.db.map_reduce0([&ctx](database& db) {
+            std::unordered_map<std::pair<sstring, sstring>, uint64_t, utils::tuple_hash> tasks;
+            return do_for_each(db.get_column_families(), [&tasks](const std::pair<utils::UUID, seastar::lw_shared_ptr<table>>& i) {
+                table& cf = *i.second.get();
+                tasks[std::make_pair(cf.schema()->ks_name(), cf.schema()->cf_name())] = cf.get_compaction_strategy().estimated_pending_compactions(cf);
+                return make_ready_future<>();
```

Comment:
We have qualified_name in schema_tables.cc. We could expose it as qualified_table_name and use it here, but I don't require that for this patch.